core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -84,7 +85,7 @@ private[spark] class TaskSetManager(
   var totalResultSize = 0L
   var calculatedTasks = 0
 
-  private val taskSetBlacklistHelperOpt: Option[TaskSetBlacklist] = {
+  private[scheduler] val taskSetBlacklistHelperOpt: Option[TaskSetBlacklist] = {
```

Comment:
yeah good question.  I wondered this myself ... I thought about instead making an `Option[(TaskSetBlacklist, BlacklistTracker)]` but it just made all the other calls ugly.  I'll make it check whether blacklistTracker is set instead.